UefiCpuPkg/Library/MmUnblockMemoryLib/MmUnblockMemoryLib.c

```diff
@@ -18,6 +18,84 @@
 #include <Library/BaseMemoryLib.h>
 #include <Library/PeiServicesLib.h>
 #include <Base.h>
+#include <PiPei.h>
+
+/**
+  This function checks if the input buffer range [UnblockBase, UnblockEnd] to unblock is
+  valid. The input range should be covered by EfiRuntimeServicesData, EfiACPIMemoryNVS or
+  EfiReservedMemoryType memory allocation HOB.
+
+  @param[in]  MemoryAllocationHob         Pointer to a memory allocation lib.
+  @param[in]  UnblockAddress              Base address of the range to unblock.
+  @param[in]  UnblockEnd                  End address of the range to unblock.
+
+  @retval RETURN_SUCCESS              The range to unblock is valid.
+  @retval RETURN_INVALID_PARAMETER    The range to unblock contains invalid memory.
+**/
+EFI_STATUS
+CheckIsUnblockRangeValid (

```

Comment:
MmUnblockMemoryLibIsUnblockableRegion()

Reply:
Changed to MmUnblockMemoryLibIsUnblockableRegion